### ISSUE TITLE
Exported JSON files persist and fill drive

### ISSUE THREAD

**Issue by schoenfbe** [heartexlabs/label-studio#3193]
**Describe the bug**
When exporting data from labelstudio (JSON format), the JSON files are saved under /label-studio/data/exports and don't seem to get deleted, which in time fills the drive with JSON files.
Those are not needed, since the export already happened.

**To Reproduce**
1. Export JSON files from labelstudio (done in a batch job, some couple 100 to 1000 per batch)
2. labelstudio keeps temporary JSON exports in its drive
3. drive is filled
4. need to manually clear label-studio/data/exports folder

**Expected behavior**
Exports should happen, but the export shouldn't be saved by labelstudio itself, or be deleted upon completion of export.

**Environment (please complete the following information):**
 - labelstudio 1.6.0 deployed in kubernetes

What do we need to change to get the desired behaviour, or is that even possible?
In conclusion, we just want labelstudio to either not generate the exports in its own drive, or delete them after the export.

Thanks for helping.

**Comment by makseq** [comment]
The easiest way to remove export files - just use a cron job with `rm`, e.g.: 
https://serverfault.com/a/256239

**Comment by makseq** [comment]
No, LS doesn't do it..
